app/questions.py

```diff
@@ -1,4 +1,5 @@
 from sympy import symbols, latex, Eq
+from . import services
```

Comment:
Could this be like, maths_service.py?
Then there are other services, db_service, hamburger_service, etc

Reply:
good call